content/rs/references/rladmin.md

```diff
@@ -1,30 +1,786 @@
+| - | - |
+| [rladmin](#use-the-rladmin-shell) | Enter the `rladmin` shell |
+| [bind](#bind) | Bind an endpoint |
+| [cluster](#cluster) | Cluster management commands |
+| [exit](#use-the-rladmin-shell)| Exit `rladmin` shell |
+| [failover](#failover) | Fail over master to replica |
+| [help](#help) | Show available commands or specific command usage |
+| [info](#info) | Show current parameters for a cluster, database, node, or proxy |
+| [migrate](#migrate) | Migrate elements between nodes |
+| [node](#node) | Node management commands |
+| [placement](#placement) | Configure shards placement policy |
+| [recover](#recover) | Recover databases |
+| [restart](#restart) | Restart database shards |
+| [status](#status) | Show status information |
+| [suffix](#suffix) | Suffix management |
+| [tune](#tune) | Tune system parameters |
+| [upgrade](#upgrade) | Upgrade the Redis or module version for a database |
+| [verify](#verify) | Cluster verification reports |
 
-## Using the rladmin CLI
+### `bind`
 
-To open the rladmin CLI:
+`rladmin bind` manages the proxy policy for the cluster or a specific database. It also can include or exclude endpoints from the proxy policy.
 
-1. Login to the RS host with an account that is a member of the **redislabs** OS group.
+```text
+rladmin bind 
+        [ db <db:id | name> ] 
+        endpoint <id> 
+        policy <single | all-master-shards | all-nodes>
+```
 
-    If you are a non-root user, you must add `/opt/redislabs/bin/` to your PATH environment variables
+| Proxy policy | Description |
+| - | - |
+| single | All traffic flows through a single proxy bound to the database endpoint (preferable in most cases)               |
+| all-master-shards | Multiple proxies, one on each master node (best for high traffic and multiple master shards)                     |
+| all-nodes | Multiple proxies, one on each node of the cluster (increases traffic in the cluster, only used in special cases) |
+| Legacy | Copies existing binding configuration from earlier versions
+
+```text
+rladmin bind 
+        [ db <db:id | name> ] 
+        endpoint <id> [ include | exclude ] 
+        proxy <id1 .. idN>
+```
+
+### `cluster`
+
+`rladmin cluster` manages cluster configuration and administration. Some `rladmin cluster` commands are only for clusters that are already configured, while others are only for new clusters that have not been configured.
+
+| Command (for configured clusters) | Description |
+| - | - |
+| `cluster certificate` | Sets the cluster certificate |
+| [`cluster config`](#cluster-config) | Updates configuration for the cluster |
+| [`cluster reset_password`](#cluster-reset_password) | Changes the password for a given email |
+| [`cluster stats_archiver`](#cluster-stats_archiver) | Enables/disables stats archiving |
+| [`cluster debug_info`](#cluster-debug_info) | Creates a support package |
+| `cluster running_actions` | Lists all active tasks |
+
+| Command (for non-configured clusters) | Description |
```

Comment:
I advise to move all these commands together with their extended description (currently lines 149-257) to a separate dedicated section about uninitialized clusters/nodes

Reply:
I'll look into this change with DOC-617